pkgs/development/python-modules/mkdocs-monorepo-plugin/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mkdocs
+, pymdown-extensions
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-monorepo-plugin";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "backstage";
+    repo = "mkdocs-monorepo-plugin";
+    rev = "c778b3010eb986a2f3b719bc7a3d29d86236c238";
+    hash = "sha256-IvQaItBglOXdXXqpklLvlxav+xeAWMdelWffY3evAX8=";
+  };
+
+  propagatedBuildInputs = [
+    mkdocs
+    pymdown-extensions
+  ];
+
+  doCheck = false;
```

Comment:
Could you comment on this, like "all tests use the network", or if possible, disable only the failing tests?